assets/js/components/GoogleChart/utils.js

```diff
@@ -0,0 +1,197 @@
+/**
+ * Google Chart utilities.
+ *
+ * Site Kit by Google, Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import set from 'lodash/set';
+import cloneDeep from 'lodash/cloneDeep';
+import merge from 'lodash/merge';
+
+/**
+ * Internal dependencies
+ */
+import { stringToDate } from '../../util/date-range/string-to-date';
+
+/**
+ * Returns the modified data for Google chart.
+ *
+ * @since n.e.x.t
+ *
+ * @param {Array|null}      data          Chart data.
+ * @param {Array|undefined} selectedStats If only certain columns should be displayed for the data set.
+ * @return {Object} The modified data to use in a chart.
+ */
+export const getGoogleChartData = ( data, selectedStats ) => {
+	// Ensure we don't filter out columns that aren't data, but are things like
```

Comment:
Seeing as we're returning unmodified data when there are no `selectedStats`, we can bail early here, and simplify the code below by removing `modifiedData` and the condition.

```suggestion
	if ( ! selectedStats?.length ) {
		return data;
	}
	// Ensure we don't filter out columns that aren't data, but are things like
```